### ISSUE TITLE
[Youtube] Video start and end time

### ISSUE THREAD

**Issue by lapinskicho** [CookPete/react-player#152]
Thanks for your work.

For Youtube player, looks like it is not taking the [end time](https://developers.google.com/youtube/player_parameters#end) from either youtubeConfig.playerVars.end or from the URL (e.g. https://www.youtube.com/v/qv6UVOQ0F44?end=40).

Also I tried the example you gave at issue #115,

```
<ReactPlayer url='https://youtu.be/qv6UVOQ0F44?start=1m33s' /> 
```
works but using youtubeConfig does not work.
```
<ReactPlayer
  url='https://youtu.be/qv6UVOQ0F44'
  youtubeConfig={{
    playerVars: {
      start: 33
    }
  }}
/>
```

But as the video end time won't work either way, a fix will be appreciated.  Thanks.

**Comment by CookPete** [comment]
I've just pushed what I thought would fix this in `0.14.2` but it looks like `start` still isn't working when put in `playerVars`. However, combining both now works for me:

```js
<ReactPlayer
  playing
  controls
  url='https://www.youtube.com/watch?v=Mh5LY4Mz15o?start=20s'
  youtubeConfig={{ playerVars: { end: 40 }}}
/>
```

I don't think `end` in the URL is supported, as it doesn't seem to work for regular youtube URLs in the same way that `start` does (eg https://www.youtube.com/watch?v=qv6UVOQ0F44&end=20)

**Comment by lapinskicho** [comment]
Thanks.

The end parameter works, but the URL needs to be like this:
```
https://www.youtube.com/v/qv6UVOQ0F44?start=30&end=31
```
or
```
https://www.youtube.com/embed/qv6UVOQ0F44?start=30&end=31
```
